### DISCUSSION TITLE
How can I define phantom using non-CT date input?

### DISCUSSION THREAD

**Discussion by Ahsong9527** [nrc-cnrc/EGSnrc#971] [Questions] [answered]
Hello,
I want to simulate that 2MeV electron beam injection on the water phantom using dosxyz,but l don't know how to define water phantom because I have not CT date. Where can l study to define water phantom non-CT date and where can l get human's skin surface data if l want
 to simulate 2MeV electron beam injection on the surface skin.
thank you

**Comment by rtownson** [comment, marked as answer]
If you look at the dosxyznrc example input files, they are using phantoms that are defined without using CT data. It is tricky to do anything complex, but if you're just looking to make a simple phantom the tools in dosxyz_gui work quite well. Just click the 'Define phantom using...' button with it set to non-CT data input. 

To model skin, you may want to define a new material based on what you can find in the literature. Or, you can define materials from density correction files and use the icrp skin material in the distribution: `pegs4/density_corrections/compounds/skin_icrp.density`

**Comment by Ahsong9527** [comment]
thanks you,sir
